lib/broken_record/job.rb

```diff
@@ -28,7 +28,7 @@ def perform
       result = BrokenRecord::JobResult.new(self)
       result.start_timer
 
-      records.each do |r|
+      records.find_each do |r|
```

Comment:
One way to mitigate the max packet size concern is to slice the ids into batches of say 1000 records using `each_slice`. I think it's worth doing now vs. waiting until later.